packages/firebase_crashlytics/android/src/main/java/io/flutter/plugins/firebase/crashlytics/firebasecrashlytics/FirebaseCrashlyticsPlugin.java

```diff
@@ -70,6 +70,15 @@ public void onMethodCall(MethodCall call, Result result) {
       exception.setStackTrace(elements.toArray(new StackTraceElement[elements.size()]));
 
       Crashlytics.setString("exception", (String) call.argument("exception"));
+
+      // Set context to show when the exception was thrown.
+      final String context = call.argument("context");
+      if (context != null) Crashlytics.setString("context", "thrown " + context);
```

Comment:
I am not sure how I feel about the "thrown " bit here. Can you explain why you went with that, as opposed to just the context?

Reply:
Because the string passed as a context ["should be in a form that will make sense in English when following the word 'thrown'"](https://api.flutter.dev/flutter/foundation/FlutterErrorDetails/context.html). The context is also only passed from `FlutterErrorDetails` and not from `runZoned`. I am not sure if every user would understand `"context": "while obtaining the image from the network"`. Everyone would probably figure it out, but adding "thrown" as a prefix seems to make it understandable more easily.

Reply:
Makes sense, but should we update iOS to match?

Reply:
Yes, I think that `"reason": "thrown while obtaining the image from the network"` sounds good. However, I am not exactly sure how it is displayed in Crashlytics. It might make sense to name the key `reason` on Android as well, as in `setString("reason", "thrown..`. What do you think?

Reply:
I'm fine with reason for now. Thanks.